rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,151 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Deploying the OSToy application with Kubernetes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+Deploy the OSToy application by creating and storing the images for the front-end and back-end microservice containers in an image repository. Then create Kubernetes deployments to deploy the application. 
```

Comment:
Both sentences here claim to involve deploying the application - "Deploy the OSToy application by..." and "...to deploy the application." Let's differentiate the actions here a little.

> Deploying an application involves creating a container image, storing it in an image repository, and defining a Deployment object that uses that image.
> 
> To deploy the sample OSToy application:
> 
> * Create the images for the front-end and back-end microservice containers
> * Store the container images in an image repository
> * Create the Kubernetes Deployment object for the application
> * Deploy the application

Reply:
Looking further through this tutorial, do we actually create and store the images anywhere? We seem to just run "oc apply" using a remote file, which uses the already created and stored `quay.io/ostoylab/ostoy-microservice:1.5.0` image. So realistically this tutorial only covers use of existing images and our intro should reflect that.